identity-service/src/routes/recaptcha.js

```diff
@@ -0,0 +1,41 @@
+const config = require('../config')
+const { handleResponse, successResponse, errorResponseForbidden } = require('../apiHelpers')
+const models = require('../models')
+
+module.exports = function (app) {
+  app.get('/captchascore', handleResponse(async (req, res, next) => {
+    if (req.headers['x-captcha-score'] !== config.get('captchaScoreSecret')) {
+      return errorResponseForbidden('Not permissioned to view captcha score.')
+    }
+
+    // TODO: make it take an array of userIds?
+    const libs = req.app.get('audiusLibs')
+    let response = {}
+    let queryParams = req.query
+
+    let userId
+    if (queryParams && queryParams.id) {
+      userId = parseInt(queryParams.id)
+    }
+
+    if (!userId) return successResponse({ msg: `userId is bad: ${userId}` })
+
+    const users = await libs.User.getUsers(
```

Comment:
i dont think we need to hit discprov in this case -- can't we just make the param wallet rather than userid? or maybe just store userid instead of wallet in the table?

Reply:
since we're moving the auth recaptcha work into the users route, this will be easier and we can query multiple userIds!